deploy/task/development/.env.example

```
@@ -0,0 +1,5 @@
+TASK_DB_HOST=shared-postgres
+TASK_DB_USER=task_user
+TASK_DB_PASSWORD=task_pass
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TASK_DB_PASSWORD key should go before the TASK_DB_USER key

(UnorderedKey)